ui/js/models/resultsets_store.js

```diff
@@ -143,11 +144,10 @@ treeherder.factory('ThResultSetStore', [
                     _.pick($location.search(), ["exclusion_profile", "visibility"]));
             }
             lastPolltime = Date.now();
-
             jobUpdatesPromise.then(function(jobList) {
+                jobList = _.flatten(jobList);
```

Comment:
I would prefer not to do that, as a 3+ deep structure would be an indication that an assumption we had about the formatting of the data was wrong, in which case we'd want what's currently happening to happen (popping up an annoying alert).